dydx/constants.py

```diff
@@ -0,0 +1,16 @@
+LIMIT_ORDERS_ADDRESS = '0xeb32d60A5cDED175cea9aFD0f2447297C125F2f4'
+SOLO_MARGIN_ADDRESS = '0x1E0447b19BB6EcFdAe1e4AE1694b0C3659614e4e'
+PAYABLE_PROXY_ADDRESS = '0xa8b39829cE2246f89B31C013b8Cde15506Fb9A76'
+WETH_ADDRESS = '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2'
+DAI_ADDRESS = '0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359'
+USDC_ADDRESS = '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48'
+
+MAX_SOLIDITY_UINT = 115792089237316195423570985008687907853269984665640564039457584007913129639935  # noqa: E501
```

Comment:
how does python handle large numbers? Is there something similar to `BigNumber` you have to use?

Reply:
Both python 2.7 and python 3 can handle arbitrarily-large integers. This is tested in the test suite which uses MAX_SOLIDITY_INT in a test for all versions of python (the ones listed in the readme)